### ISSUE TITLE
Sonarr downloading both NZB and Torrents of same file

### ISSUE THREAD

**Issue by antigravity83** [Sonarr/Sonarr#2893]
Sonarr will download a file via Torrent (usually because its available first), then it downloads the exact same NZB shortly after. Meaning I download most files twice.

**Cleaning.Up.S01E02.1080P.HDTV.x264-MTB downloaded via Torrent at 8:54am**
![image](https://user-images.githubusercontent.com/27241813/51285611-3ec4fa80-1a3b-11e9-9781-36e5b1063309.png)

**Same download via NZB is added - but queued to download at 9:54am**
![image](https://user-images.githubusercontent.com/27241813/51285654-6d42d580-1a3b-11e9-8959-19dfa2af9cba.png)

**Here are my delay profiles - both are set to 10 minutes. Current time is 9:39am, so why would the NZB above be queued to download at 9:54am**
![image](https://user-images.githubusercontent.com/27241813/51285688-8ea3c180-1a3b-11e9-9ebf-7193b8983e81.png)

Logs: [sonarr.txt](https://github.com/Sonarr/Sonarr/files/2766550/sonarr.txt)

 - Sonarr Version: 2.0.0.5298
 - Operating System: Ubuntu 16.04
 - Mono: 5.12.0.301

**Comment by markus101** [comment]
The imported file is still in the series folder?
What is your RSS Sync Interval set to?

We're going to need debug logs (as requested) of the RSS sync that added the pending release to the queue to determine why it was added to to pending releases.

**Comment by antigravity83** [comment]
> The imported file is still in the series folder?
Yes, the first file that was downloaded was added to the series folder once it downloaded. (If I understood your question correctly)

So the original file was showing as downloaded in the main series page - but it still added the exact same file via NZB (not a repack or anything). 

> What is your RSS Sync Interval set to?
60 minutes

> We're going to need debug logs (as requested) of the RSS sync that added the pending release to the queue to determine why it was added to to pending releases.
I've changed the log to debug. Will provide an update when the issue occurs with a different file.

**Comment by markus101** [comment]
Which download client are you using for torrents?

Is the NZB actually added to the usenet client? The pending queue doesn't mean it's going to be grabbed, just means it could be grabbed if that release is still the best release available the next time the RSS Sync runs. Givent he 60 minute interval it looks like the torrent was grabbed at 8:54 and at the same time the NZB was added to the pending queue and at 9:54 the pending release would be removed because there is already a file of the same quality on disk.

It's also possible that Sonarr thought the torrent failed to be added to your torrent client and queued the NZB for later, but the debug logs will show that.

**Comment by antigravity83** [comment]
> Which download client are you using for torrents?

I'm using ruTorrent.

> Is the NZB actually added to the usenet client? The pending queue doesn't mean it's going to be grabbed, just means it could be grabbed if that release is still the best release available the next time the RSS Sync runs. Givent he 60 minute interval it looks like the torrent was grabbed at 8:54 and at the same time the NZB was added to the pending queue and at 9:54 the pending release would be removed because there is already a file of the same quality on disk.

Ahh ok, I see. I'll see if the second file actually gets downloaded.

> It's also possible that Sonarr thought the torrent failed to be added to your torrent client and queued the NZB for later, but the debug logs will show that.

Will provide debug logs once it happens again - thanks for your support

**Comment by markus101** [comment]
It'll be like another other rejected release in an RSS Sync, release rejected for: <list of reasons>

**Comment by markus101** [comment]
Are duplicates being downloaded? Or can we close this?

**Comment by antigravity83** [comment]
Release rejected for the following reasons: [Permanent] Quality for existing file on disk is of equal or higher preference: WEBDL-720p v1

Seeing a lot of the above - so I'm guessing whilst files are appearing in the delayed download activity list, they aren't actually getting downloaded. 

That was my misunderstanding - thanks for your help
